spec/components/lookup_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "spec_helper"
+
+describe "lookup", type: :view do
+  def component_name
+    "lookup"
+  end
+
+  def render_component(locals)
+    if block_given?
+      render("components/#{component_name}", locals) { yield }
+    else
+      render "components/#{component_name}", locals
+    end
+  end
+
+  it "displays external link" do
+    render_component(link: { text: "Find a postcode on Royal Mail", href: "royalmail.com/share" })
+    assert_select ".govuk-link--external"
```

Comment:
Yes, so it was suggested to test for the conditional inclusion of an external (postcode lookup) link.  The actual partial includes a new class `.govuk-link--external` should this link be included.  However this might be too granular as per `59`